onboarding/fibonacci.py

```diff
@@ -1,18 +1,30 @@
 def fibonacci_term(n):
-    '''
-    Finds nth term of the Fibonacci sequence from:
-        Term | value
-        ------------
-          1  |   1
-          2  |   1
-          3  |   2
-          4  |   3
-          5  |   5
-         ... |  ...
+  '''
+  Finds nth term of the Fibonacci sequence from:
+       Term | value
+       ------------
+         1  |   1
+         2  |   1
+         3  |   2
+         4  |   3
+         5  |   5
+        ... |  ...
 
-    @params n: integer
-    @returns integer, nth term of the Fibonacci sequence
-    '''
+  @params n: integer
+  @returns integer, nth term of the Fibonacci sequence
+  '''
 
-    # insert code here
-    pass
+  # insert code here
+  if n == 0:
+    return 0
+  elif n == 1:
+    return 1
+  else:
+    n1 = 0
+    n2 = 1
+    for i in range(2,n):
```

Comment:
I think you want to take your loop till `n + 1` since it doesn't give the correct output right now.